content/posts/2025-01-28-cli_use_cases.md

```diff
@@ -0,0 +1,365 @@
+
+All together, the command to run flow capture with all the features on our pod IP will be:
+```sh
+oc netobserv flows --enable_all --peer_ip=10.131.0.19
+```
+
+The script will connect to your cluster and start deploying eBPF agents and collector pod:
+```sh
+Checking dependencies... 
+'yq' is up to date (version v4.43.1).
+'bash' is up to date (version v5.2.26).
+Setting up... 
+cluster-admin
+creating netobserv-cli namespace
+namespace/netobserv-cli created
+creating service account
+serviceaccount/netobserv-cli created
+clusterrole.rbac.authorization.k8s.io/netobserv-cli unchanged
+clusterrolebinding.rbac.authorization.k8s.io/netobserv-cli unchanged
+creating collector service
+service/collector created
+creating flow-capture agents:
+opt: pkt_drop_enable, value: true
+opt: dns_enable, value: true
+opt: rtt_enable, value: true
+opt: network_events_enable, value: true
+opt: udn_enable, value: true
+opt: pkt_xlat_enable, value: true
+opt: filter_peer_ip, value: 10.129.0.19
+daemonset.apps/netobserv-cli created
+Waiting for daemon set "netobserv-cli" rollout to finish: 0 of 2 updated pods are available...
+Waiting for daemon set "netobserv-cli" rollout to finish: 1 of 2 updated pods are available...
+daemon set "netobserv-cli" successfully rolled out
+Running network-observability-cli get-flows... 
+pod/collector created
+pod/collector condition met
+```
+
+Once that done, it will connect to the collector and display its output:
+```sh
+------------------------------------------------------------------------
+         _  _     _       _                       ___ _    ___
+        | \| |___| |_ ___| |__ ___ ___ _ ___ __  / __| |  |_ _|
+        | .' / -_)  _/ _ \ '_ (_-</ -_) '_\ V / | (__| |__ | | 
+        |_|\_\___|\__\___/_.__/__/\___|_|  \_/   \___|____|___|
+
+------------------------------------------------------------------------
+INFO[0000] Log level: info
+Option(s): enable_all|peer_ip=10.129.0.19 
+INFO[0000] Kernel version: 5.14.0-427.50.1.el9_4.x86_64 
+INFO[0000] Starting Flow Capture...                     
+INFO[0000] Creating database...                         
+INFO[0000] flows.db created                             
+INFO[0000] Create flows table...                        
+INFO[0000] flows table created 
+```
+
+At this stage, the collector wait for incoming data. If nothing shows yet, it means that no traffic is captured. Try to open the route of your application or update the filters of the capture.
+
+Once some traffic is captured, the output will look like:
+![cli network events]({page.image('cli/connectivity-scenario-cli-events.png')})
+
+Cycle to the **network events** view. In this capture, we see that the traffic is blocked by a network policy since it reports the `NetpolNamespace` event.
```

Comment:
[This PR](https://github.com/netobserv/network-observability-cli/pull/135) should provide a better output for the network events

Reply:
Awesome ! Do you have a screen ?
Else I will spin up a cluster

Reply:
not yet, cluster not started yet